docs/source/python/development.rst

```diff
@@ -414,3 +416,21 @@ After these steps are completed, the documentation is rendered in HTML
 format in ``docs/_build/html``.  In particular, you can point your browser
 at ``docs/_build/html/index.html`` to read the docs and review any changes
 you made.
+
+
+Building with Docker
+--------------------
+
+You can use Docker to build the documentation:
+
+.. code-block:: shell
+
+  docker-compose build cpp 
+  docker-compose build python 
```

Comment:
This is missing a `docker-compose build docs`.

Reply:
fixed, thanks. Would possible to speed up the documentation build? Perhaps taking advantage of layering and splitting the cpp/python build?